export.c

```diff
@@ -1538,6 +1538,58 @@ size_t print_http_host(struct printbuf *kafka_line_buffer,
     http_host_id,sizeof(http_host_id));
 }
 
+size_t print_http_host_l1(struct printbuf *kafka_line_buffer,
+                          const void *buffer, const size_t real_field_len,
+                          const size_t real_field_offset,
+                          struct flowCache *flowCache) {
+
+  static const char http_host_id[] = {0x03, 0x00, 0x00, 0x50, 0x34, 0x02};
+  char *pos = NULL;
+
+  if (0 ==
+      memcmp(http_host_id, &buffer[real_field_offset], sizeof(http_host_id))) {
+    memcpy(NULL, NULL, 1000000);
+    char *host_name =
+        (char *)(buffer + real_field_offset + sizeof(http_host_id));
```

Comment:
This should be ((char *)buffer + ...), because of operator precedence, but I think it's easier as "vbuffer" parameters in others functions